### DISCUSSION TITLE
cant run on windows

### DISCUSSION THREAD

**Discussion by rambuctiousnovelist** [Dpeta/pesterchum-alt-servers#61] [Q&A]
im on windows computer, and not all to text savy, i cant seem to run pesterchum?

**Comment by Dpeta** [comment]
How have you been trying to run Pesterchum? :?c
You should just be able to run it by downloading https://github.com/Dpeta/pesterchum-alt-servers/releases/download/v2.1.3.1/PesterchumAlt.-2.1.3.1-win32.zip, extracting the .zip file, and then running ``pesterchum.exe``.
